cmake/FindgRPC.cmake

```diff
@@ -65,6 +65,13 @@ endif ()
 # gRPC always requires Thread support.
 find_package(Threads REQUIRED)
 
+# ~~~
+# gRPC sometimes requires opentelemetry-cpp::api but it neglects to
+# say `find_dependency(opentelemetry-cpp)`:
+#    https://github.com/grpc/grpc/issues/36624
+# ~~~
+find_package(opentelemetry-cpp CONFIG QUIET)
```

Comment:
just to be clear, if NOT `opentelemetry-cpp_FOUND`, this will not error, nor will it even log an error, right? (which is what we want)

Reply:
Yup, if OpenTelemetry does not exist, then gRPC is (unlikely) to require it, and this workaround is not needed.  Being silent and getting `NOT opentelemetry-cpp_FOUND` is the outcome we want in that case.